### ISSUE TITLE
Running on a Windows host 

### ISSUE THREAD

**Issue by ailintom** [rnsrk/docker-wisski#2]
The commit 3f0b768da5f737fb57b492160b648a52de067a45 has replaced port-forwarding with a network bridge. This creates problems for running the package on a Windows host, because the The docker (Linux) bridge network is not reachable from the Windows host out of the box. 
Would not it be possible to revert to port-forwarding, by reinstating lines 10-11: 
```
    ports: 
      - "8081:80"
```

**Comment by dgoldhahn** [comment]
I agree that using port forwarding will be easier to handle for most docker users. Especially since establishing a network bridge might be new to many users and is not covered in the readme.
